make/production/jumpstart-personal.make

```diff
@@ -1,5 +1,7 @@
 core = 7.x
 api = 2
+; Default directory should be sites/all/modules/contrib
+defaults[projects][subdir] = contrib
```

Comment:
This project has fallen so far behind it is not worth supporting any further. I would avoid updates to it in the future.